_includes/scenarios/integrate/create-attribute-for-yourself.md

```diff
@@ -0,0 +1,99 @@
+This guide explains the end to end flow of creating an [Attribute]({% link _docs_integrate/data-model-overview.md %}#attributes) for your own Connector as its Integrator. As there are [IdentityAttributes]({% link _docs_integrate/data-model-overview.md %}#identityattribute) and [RelationshipAttributes]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute), a distinction must be made between these two types of Attributes when creating an Attribute for yourself.
+
+## Create an IdentityAttribute for yourself
+
+This section is about how to create an IdentityAttribute for your own Connector that is not initially shared with any other Identity. From a technical point of view, this corresponds to the creation of a [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) whose `content` is given by the IdentityAttribute that is intended to be created and whose `shareInfo` is undefined. As usual, such a LocalAttribute is referred to as a **RepositoryAttribute**.
+
+Since knowledge about IdentityAttributes is required in the following, you should take a look at our IdentityAttribute introduction before you continue reading this guide. In particular, a description of the two kinds of IdentityAttributes, the simple IdentityAttributes and the complex IdentityAttributes, can be found there.
+{: .notice--info}
+
+<!--- TODO: Insert link to "IdentityAttribute introduction" --->
+
+### Input for creating a RepositoryAttribute
+
+To create a RepositoryAttribute, proceed as described in the [Create a RepositoryAttribute]({% link _docs_use-cases/use-case-consumption-create-a-repositoryattribute.md %}) use case documentation. As input for the creation of a RepositoryAttribute, the following `content` must be used:
+
+```jsonc
+{
+  "content": {
+    "validFrom": "<start of IdentityAttribute's validity>",
+    "validTo": "<end of IdentityAttribute's validity>",
+    "value": {
```

Comment:
I understand your point of view. At this point, I'd prefer to use the order of the properties as it is in the Data Model Overview for the IdentityAttribute to be consistent.

Reply:
Then how about adjusting it in the Data Model Overview, too, such that the required properties are on top?

Reply:
I think the order of the properties in the Data Model Overview comes from how the IdentityAttribute is defined with the help of the AbstractAttribute in the content library. From this perspective, the current order in the Data Model Overview makes sense to me. However, I also think that it would be a legitimate order of the properties to sort by relevance (for example, required or optional). IMO we should have to take a look at this for the other data object types as well and then make a standardised decision.